source/jormungandr/jormungandr/interfaces/v1/serializer/geocode_json.py

```diff
@@ -161,6 +163,22 @@ def get_properties(self, obj):
         return {p.get("key"): p.get("value") 
                 for p in obj.get('properties', {}).get('geocoding', {}).get('properties', [])}
 
+    def get_address(self, obj):
+        address = obj.get('properties', {}).get('geocoding', {}).get('address', None)
+        if not address:
+            return None
+
+        return {
```

Comment:
This is duplicated it seems (close from https://github.com/CanalTP/navitia/pull/2300/files#diff-204c59b4be2984c009079df1061d66f7R131and maybe even from https://github.com/CanalTP/navitia/pull/2300/files#diff-204c59b4be2984c009079df1061d66f7R180 )